gdx/src/com/badlogic/gdx/scenes/scene2d/utils/TiledDrawable.java

```diff
@@ -78,5 +83,17 @@ public void draw (Batch batch, float x, float y, float width, float height) {
 				x += regionWidth;
 			}
 		}
+		batch.setColor(batchColor);
+	}
+
+	/** Creates a new drawable that renders the same as this drawable tinted the specified color. */
+	public TiledDrawable tintTiled (Color tint) {
+		TiledDrawable drawable = new TiledDrawable(this);
+		drawable.tint = tint.toFloatBits();
+		drawable.setLeftWidth(getLeftWidth());
+		drawable.setRightWidth(getRightWidth());
+		drawable.setTopHeight(getTopHeight());
+		drawable.setBottomHeight(getBottomHeight());
+		return drawable;

```

Comment:
Sprite knows how to draw a region while preserving any trimmed whitespace. This is why a SpriteDrawable is used if the region is an AtlasRegion. TextureRegionDrawable#tint could probably return a Drawable, but I'm not sure overloading `tint` buys us much (next we'd have a Tintable interface, which just doesn't add much to the API).